code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -196,3 +203,223 @@
 			if(C.eye_blurry < 60)
 				C.AdjustEyeBlurry(10)
 				visible_message("<span class='danger'>[src] buries its fangs deep into the [inject_target] of [target]!</span>")
+
+
+/mob/living/simple_animal/hostile/headcrab/gonarch
+	name = "gonarch"
+	desc = "This is a... headcrab? Looks more like a walking tank."
+	icon = 'icons/mob/gonarch.dmi'
+	icon_state = "gonarch"
+	icon_living = "gonarch"
+	icon_dead = "gonarch"
+	pixel_x = -16
+	health = 800
+	maxHealth = 800
+	force_threshold = 16
+	dodging = 1
+	gender = FEMALE //doesn't matter though
+	melee_damage_lower = 45
+	melee_damage_upper = 75
+	ranged = 0
+	move_to_delay = 10
+	speed = 1.5
+	attacktext = "slashes"
+	attack_sound = 'sound/creatures/gonarch_attack.ogg' //TODO maybe find a better sound. Something slashing maybe.
+	speak_emote = list("hisses")
+	robust_searching = 1
+	obj_damage = 200
+	stat_attack = CONSCIOUS // They don't care for killing. They want their peace.
+	armour_penetration = 5
+	attack_sound = 'sound/effects/blobattack.ogg'
+	death_sound = 'sound/creatures/gonarch_die.ogg'
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	stop_automated_movement_when_pulled = 0
+	move_resist = MOVE_FORCE_EXTREMELY_STRONG
+	environment_smash = ENVIRONMENT_SMASH_RWALLS
+	vision_range = 9
+	aggro_vision_range = 9
+	pass_flags = PASSTABLE | PASSMOB //It is so big, you can walk through its legs.
+	density = FALSE //Same as above.
+	//It has so many vars, to enable admins to change its behaviour on the fly. This is also the reason why its statemachine uses a lot of proc calls, so admins can call these manually.
+	can_zombify = FALSE
+	var/gonarch_standard_vision_range = 9
+	var/gonarch_aggro_vision_range = 9
+	var/gonarch_standard_speed = 2
+	var/area/home_nest_area
+	var/turf/home_nest_turf
+	var/homesick = 0
+	var/mode = MODE_REST
+	var/list/children = list()
+	var/desired_child_count = 10
+	var/time_last_babies = 0
+	var/list/path_to_home
+	var/frustration = 0
+	var/frustration_limit = 10
+	var/gonarch_nestfind_range = 8
+	var/gonarch_rampage_trigger = 30
+	var/gonarch_screech_range = 8
+	var/gonarch_finding_home_limit_attempts = 2
+	var/gonarch_finding_home_attempts = 0
+	var/gonarch_headcrab_spawn_frequency = 600
```

Comment:
May want to [document each and every one of these variables](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine)